File: practice_java_program/notpad/LeadersElement.java
public class LeadersElement{
	public static void main(String[] args){
		int[] numbers = {16, 17, 4,3,5,2}; //Leaders element 17, 5, 2
		int maxFromRight = numbers[numbers.length-1];
		System.out.print("Leaders Elements = "+  maxFromRight);
		for(int i=numbers.length-2; i>=0; i--){
			if(numbers[i]>maxFromRight){
				maxFromRight=numbers[i];
				System.out.print(", "+maxFromRight);
			}
		}
	}
}

File: python_oriented_interview/notpad/LeadersElement.py
numbers = [16, 17, 4, 3, 5, 2]
maxFromRight = numbers[-1]
print(f"Leaders Elements = {maxFromRight}", end="")
for i in range(len(numbers) - 2, -1, -1):
    if numbers[i] > maxFromRight:
        maxFromRight = numbers[i]
        print(f", {maxFromRight}", end="")
print()
